onnxruntime/python/tools/quantization/fusions/fusion_gelu.py

```diff
@@ -173,11 +175,9 @@ def fuse_2(
             if not self.has_constant_input(sqrt_node, 2.0):
                 return False
 
-        root_node = self.model.get_parent(div, 0, output_name_to_node)
-        if root_node is None:
-            return False
```

Comment:
Note: this change allows the fusion to also happen at the graph's input. Without this change, the fusion only happens in the middle of the graph.